packages/x-components/src/components/result/base-result-image.vue

```diff
@@ -208,14 +208,13 @@
 
 <style lang="scss" scoped>
   .x-result-picture {
+    // the loaderStyles prop is positioning absolutely to this container
+    position: relative;
     min-width: 1px;
     min-height: 1px;
-    position: relative;
 
     &__image {
-      max-width: 100%;
       max-height: 100%;
-      object-fit: contain;
```

Comment:
`object-fit` gets applied with the `x-picture  img` styles coming from the new XDS